frontends/web/src/contexts/KeystoreContext.tsx

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright 2024 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+*
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { createContext } from 'react';
+import { AccountCode } from '../api/account';
+
+type KeystoreContextProps = {
+    requestKeystore: (accountCode: AccountCode, onSuccess?: () => void, onCancel?: () => void) => Promise<void>;
```

Comment:
since now the backend does half the work and the frontend does half the work in managing this, it could be useful to add some docstrings in these two functions about how to use them and what they do exactly in each case.

also useful to document what happens if one does *not* call this function - the backend calls `keystore, err := account.Config().ConnectKeystore()` anyway which will generally work, but without the benefits introduced in this PR.